SPECS/maven-compiler-plugin/maven-compiler-plugin.spec

```diff
@@ -0,0 +1,222 @@
+%bcond_without bootstrap
+Summary:        Maven Compiler Plugin
+Name:           maven-compiler-plugin
+Version:        3.8.1
```

Comment:
why not using latest (3.11.0)?

Reply:
bootstrap supports 3.8.1.
The build fails with newer versions.